packages/cells/src/widget.ts

```diff
@@ -166,7 +170,7 @@ const RENDER_TIMEOUT = 1000;
  * A base cell widget.
  */
 export
-class Cell extends Widget implements IRenderMime.IReadyWidget {
+class Cell extends Widget implements DocumentRegistry.IReadyWidget {
```

Comment:
I am a bit surprised that cell depends on the document registry. At least conceptually, cells shouldnn't know anything about documents. The `IReadyWidget` is really simply, can we put it somewhere else that cell can depend on?